ci/assets/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
         API_PORT: '8081'
         DEBUG: 'false'
         LOG_LEVEL: 'info'
-        STATSD_DSN: 'statsd:8125'
+        STATSD_DSN: ''
```

Comment:
I prefer to remove it if it's not gonna be used. Also the behaviour of statsd should be explicit mentioned on the docs.